src/validators/satelliteIdValidator.ts

```diff
@@ -0,0 +1,12 @@
+import { Request, Response, response } from 'express'
+import { assert, pattern, string } from 'superstruct'
+
+export const satelliteIdRouteValidation = (req: Request, res: Response, next: CallableFunction): void | ReturnType<typeof response.sendStatus> => {
```

Comment:
Is there id route? Also please start function names with verbs. 
So `validateSatelliteIdRoute`. That's because functions are actions, it's easier to read them later on. 
You could write here `validateGetTwoLinesRequest` or `validateRequestWithSatelliteId`.